src/components/ToastNotification.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <transition
+    enter-active-class="duration-500 ease"
+    enter-from-class="transform opacity-0"
+    leave-active-class="duration-500 ease"
+    leave-to-class="transform opacity-0"
+  >
+    <div
+      v-if="isActive"
+      class="rounded p-4 mb-2 bg-red-500 text-white"
+    >
+      <span>{{ text }}</span>
+      <ul v-if="listItems?.length">
+        <li
+          v-for="item in listItems"
+          :key="item"
+        >
+          {{ item }}
+        </li>
+      </ul>
+      <button
+        v-if="dismissButtonText"
+        class="block rounded bg-slate-200/50 py-1 px-2 mt-2"
```

Comment:
you could add a hover effect by settings `hover:bg-something transition-colors duration-75`

Reply:
Good idea 👌